plenum/test/consensus/order_service/test_send_master_reordered.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from plenum.common.messages.internal_messages import MasterReorderedAfterVC
+
+PREV_VIEW_PP_CERT = 10
+
+
+@pytest.fixture(params=[(0, 5), (1, 10), (2, 15)])
+def view_pp(request):
+    return request.param
+
+
+def test_send_master_reordered(orderer, view_pp):
+    view_no, pp_seq_no = view_pp
+    orderer._data.prev_view_prepare_cert = PREV_VIEW_PP_CERT
+    orderer._add_to_ordered(view_no, pp_seq_no)
+    if pp_seq_no == PREV_VIEW_PP_CERT:
```

Comment:
It needs to be checked that we don't send the message in `else` case. It would be better to add the expected result (sent MasterReorderedAfterVC or not, True or False) into  `view_pp` fixtures params:
```
params=[((0, 5), False),
               ((1, 9), False),
               ((1, 10), True),
               ((1, 11), True),
               ((1, 15), True)
]
```